src/app/[locale]/(marketing)/blog/[slug]/page.tsx

```
@@ -0,0 +1,85 @@
+                    {post.frontmatter.category && (
+                        <span className="mb-2 inline-block rounded-sm bg-primary-1/20 px-2 py-0.5 text-xs font-semibold">
+                            {post.frontmatter.category}
+                        </span>
+                    )}
+                    <h1 className="text-3xl font-bold md:text-4xl">{post.frontmatter.title}</h1>
+                    <p className="mt-2 text-gray-600">{post.frontmatter.description}</p>
+                    <time className="mt-3 block text-sm text-gray-400">{post.frontmatter.date}</time>
+                </header>
+                <article
+                    className="prose prose-lg prose-headings:font-bold prose-a:text-black prose-a:underline prose-pre:border prose-pre:border-n-1 prose-pre:bg-white max-w-none"
+                    dangerouslySetInnerHTML={{ __html: post.html }}
```

Biome (2.3.14)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)